interfaces/udev/spec.go

```diff
@@ -93,6 +108,14 @@ func (c byTagAndSnippet) Less(i, j int) bool {
 
 // Snippets returns a copy of all the snippets added so far.
 func (spec *Specification) Snippets() (result []string) {
+	// If the interface controls it's own device cgroup, then
```

Comment:
s/the interface/one of the interfaces/ , spec is used over all the plugs and slots of a snap.

Reply:
Done